gulpfile.js

```diff
@@ -0,0 +1,24 @@
+const del = require('del');
+const eslint = require('gulp-eslint');
+const gulp = require('gulp');
+const zip = require('gulp-zip');
+
+gulp.task('build', ['lint', 'clean:dist'], function () {
+  return gulp.src('src/*')
```

Comment:
@pdehaan: hmm, building the XPI doesn't recurse into subdirectories. I think this line needs to be
```
gulp.src('src/**/*')
```